### PR TITLE
[WIP] Setting up Test Environment

### PR DESCRIPTION
**Pull request by deleugpn**
This will help start writing tests, but unfortunately I hit a wall: Our factory relies on the existence of `App\Users` in a Laravel application. We need to have the `User` Model within the package as well as think about how to handle the fact that somebody might delete the users migration that comes with Laravel by default.

### CODE REVIEW

File: tests/Integration/PostTest.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+namespace AndreasElia\Tests\Integration;
+
+use AndreasElia\Forum\Models\Post;
+use AndreasElia\Tests\IntegrationTestCase;
+
+class PostTest extends IntegrationTestCase
+{
+
+    public function testCreatePost()
```
Comment (deleugpn) [inline tests/Integration/PostTest.php]: This test is useless right now, I know, but I'm using it to go through the settings of the tests. Once I have this test passing, we can remove it and start working on actual tests.

**Comment by AndreasElia** [comment]
Looks good so far, please bear in mind StyleCI.

**Comment by deleugpn** [comment]
Why was this closed?

**Comment by AndreasElia** [comment]
@deleugpn change your target branch to master, I removed my develop branch.

**Pull request closed without merging**